src/Core/src/Sts/Input/AssumeRoleRequest.php

```diff
@@ -260,6 +260,22 @@ public function __construct(array $input = [])
         parent::__construct($input);
     }
 
+    /**
+     * @param array{
+     *   RoleArn?: string,
+     *   RoleSessionName?: string,
+     *   PolicyArns?: PolicyDescriptorType[],
+     *   Policy?: string,
+     *   DurationSeconds?: int,
+     *   Tags?: Tag[],
+     *   TransitiveTagKeys?: string[],
+     *   ExternalId?: string,
+     *   SerialNumber?: string,
+     *   TokenCode?: string,
+     *   SourceIdentity?: string,
+     *   '@region'?: string|null,
+     * }|AssumeRoleRequest $input
+     */
     public static function create($input): self
```

Comment:
This method is just a proxy for `--construct`.
I'm not sure duplicating this docbloc provides any value.

If the user enters invalid content, the STA should report the issue when calling `new self($input);` right?